apps/web/app/routes/examples/forms/multiple-forms.tsx

```diff
@@ -0,0 +1,146 @@
+import hljs from 'highlight.js/lib/common'
+import type {
+  ActionFunction,
+  LoaderFunction,
+  MetaFunction,
+} from '@remix-run/node'
+import { formAction } from '~/formAction'
+import { z } from 'zod'
+import Form from '~/ui/form'
+import { metaTags } from '~/helpers'
+import { makeDomainFunction } from 'domain-functions'
+import Example from '~/ui/example'
+
+const title = 'Multiple Forms'
+const description =
+  'In this example, we show how you can handle multiple forms in the same route.'
+
+export const meta: MetaFunction = () => metaTags({ title, description })
+
+const code = `const loginSchema = z.object({
+  _action: z.enum(['/login', '/contact']),
+  email: z.string().email(),
+  password: z.string().min(8),
+})
+
+const contactSchema = z.object({
+  _action: z.enum(['/login', '/contact']),
+  email: z.string().email(),
+  message: z.string().min(1),
+})
+
+const loginMutation = makeDomainFunction(loginSchema)(
+  async (values) => values
+)
+
+const contactMutation = makeDomainFunction(contactSchema)(
+  async (values) => values
+)
+
+export const action: ActionFunction = async ({ request }) => {
+  const formData = await request.clone().formData()
+  const action = formData.get('_action')
+
+  if (action === '/login') {
+    return formAction({
+      request: request.clone(),
+      mutation: loginMutation,
+      schema: loginSchema,
+    })
+  }
+
+  if (action === '/contact') {
+    return formAction({
+      request: request.clone(),
+      mutation: contactMutation,
+      schema: contactSchema,
+    })
+  }
+}
+
+export default () => {
+  return (
+    <Example title={title} description={description}>
+      <Form
+        schema={loginSchema}
+        buttonLabel="Login"
+        hiddenFields={['_action']}
+        values={{ _action: '/login' }}
+      />
+      <Form
+        schema={contactSchema}
+        multiline={['message']}
+        buttonLabel="Contact Us"
+        hiddenFields={['_action']}
+        values={{ _action: '/contact' }}
+      />
+    </Example>
```

Comment:
Also we should replace `<Example>` tag with a fragment since it is the code to render the example and not a part of it.
```suggestion
    <>
      <Form
        schema={loginSchema}
        buttonLabel="Login"
        hiddenFields={['_action']}
        values={{ _action: '/login' }}
      />
      <Form
        schema={contactSchema}
        multiline={['message']}
        buttonLabel="Contact Us"
        hiddenFields={['_action']}
        values={{ _action: '/contact' }}
      />
    </>
```